### PR TITLE
Homework start page

### PR DESCRIPTION
**Pull request by PoMaksym**
trying to build a framework

### CODE REVIEW

**Review by QALightLecorPython — APPROVED** [review]

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 /venv/
 /.pytest_cache/
 .idea
-/test_start_page.py
+/test/test_start_page.py
```
Comment (QALightLecorPython) [inline .gitignore:4]: Why do you ignoring tests ?

File: pages/search_page.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from constants.search import SearchConsts
+from pages.base_page import BasePage
+from pages.header import Header
+
+
+class SearchPage(BasePage):
```
Comment (QALightLecorPython) [inline pages/search_page.py:6]: Does search is page ? I think is a part of the Header

File: pages/start_page.py [hunk 3]
```diff
@@ -42,17 +42,35 @@ def sign_up(self, username, email, password):
...
+        """Click sign up and verify"""
+        # Click button
+        self.click(self.constants.SIGN_UP_BUTTON_XPATH)
+        assert not self.is_exists(self.constants.SIGN_UP_BUTTON_XPATH)
 
     def verify_success_sign_up(self, username):
         """Verify success Sign Up using hello message"""
         username = username.lower()
-        assert self.get_element_text(self.constants.HELLO_MESSAGE_XPATH) == self.constants.HELLO_MESSAGE_TEXT.format(
+        assert self.get_element_text(self.constants.H) == self.constants.HELLO_MESSAGE_TEXT.format(
```
Comment (QALightLecorPython) [inline pages/start_page.py:71]: `H` ?

**Review by QALightLecorPython — CHANGES_REQUESTED** [review]

File: pages/search_page.py (hunk 2 above)
Comment (PoMaksym) [reply pages/search_page.py:6]: Got it,  i'll fix it

**Pull request closed without merging**